README.md

```diff
@@ -372,6 +373,20 @@ Available `options`:
 * `ttr` - time to release in seconds. The time after which, if there is no active
 connection in the session, it will be released with all its tasks.
 
+## Session identify
+
+```lua
+queue.identify(session_uuid)
+```
+In the queue the session has a unique UUID and one session can have many
+connections. Also, the consumer can reconnect to the existing session during the
```

Comment:
> one session can have many connections

I would say 'many connections may share one logical session' or kinda. It is a thing of taste, feel free to ignore.

Reply:
Accepted.